tactic/default.lean

```diff
@@ -4,4 +4,5 @@ import
   tactic.finish
   tactic.mk_iff_of_inductive_prop
   tactic.wlog
+  tactic.split_ifs
```

Comment:
How about adding the file reference to `tactic.inductive`? That way you don't need to import it elsewhere. The tactic is large enough to deserve its own file but not so large that it can't be part of the standard set of tactics.

Reply:
There is no `tactic.inductive`, just `tactic.interactive`.  And I just moved the import there.